docs/workspaces.md

```diff
@@ -37,27 +37,50 @@ information, see [Resource Persistence](./templates/resource-persistence.md).
 
 When a workspace is deleted, all of the workspace's resources are deleted.
 
-## Workspace scheduling
+## Workspace auto-stop
 
-By default, workspaces are manually turned on/off by the user. However, a schedule
-can be defined on a per-workspace basis to automate the workspace start/stop.
+In Coder, workspaces can auto-stop when the user is not actively using the workspace. By default, Coder recognizes the following actions as activity and will keep the workspace online:
 
-![Scheduling UI](./images/schedule.png)
+- Open SSH connections (VS Code Extension, JetBrains Gateway, `coder ssh`)
+- Open tunnels (`coder port-forward`)
+- Open WebSockets (web terminal, [web IDEs](./ides/web-ides.md) such as code-server and JupyterLab)
 
-### Autostart
+> [Custom activity](#TODO) can be registered via Coder's REST API, such as an ongoing data science pipeline.
 
-The autostart feature automates the workspace build at a user-specified time
-and day(s) of the week. In addition, users can select their preferred timezone.
+<!-- TODO. Document this. We may want to move to a custom activity section. See coder/coder[#8064](https://github.com/coder/coder/issues/8064) and coder/coder#5629 -->
 
-![Autostart UI](./images/autostart.png)
+Your workspace's auto-stop behavior can be seen in the dashboard. Template settings in Coder determine if users can alter their workspace schedules. 
 
-### Autostop
+![TODO: use real screenshot](./images/workspaces/stops-after-3-hours.png)
 
-The autostop feature shuts off workspaces after given number of hours in the "on"
-state. If Coder detects workspace connection activity, the autostop timer is bumped up
-one hour. IDE, SSH, Port Forwarding, and coder_app activity trigger this bump.
+### Max workspace lifetime (enterprise)
 
-![autostop UI](./images/autostop.png)
+Workspaces may also have a max lifetime, which enforces a stop or restart even if you are actively using your workspace.
+
+![TODO: use real screenshot](./images/workspaces/approaching-stop.png)
+
+#### Quiet hours
+
+To avoid interruptions, users can establish their "quiet hours", which are by default set from midnight to 4 AM, during which these actions occur. Users can also extend the deadline by up to 3 hours if they are in the middle of important work.
```

Comment:
"midnight to 4 AM", but what time zone?

Reply:
Also.. should we bother allowing bumps other than 3 hours? This seems like a good opportunity for us to reduce the number of decisions the user has to make.

Reply:
Yeah, should probably just be a single bump with a duration of 3 hours.

Reply:
It's odd to me how the bump is 3 hours but the quiet hour window is 4 hours. There are also 8 hours in a work day, which isn't divisible by 3. I think the bump should be either 2 or 4 hours.

Reply:
2-hour bump makes sense to me.

Reply:
What would be the case when developers are distributed around different TZ? 12-4 AM in PST TZ isn't an excellent quiet hours window for someone in Sydney.

Reply:
The site sets a default time and timezone but users can set their own time and timezone which overrides the default for their own workspaces.

Reply:
Is the TZ in the DB? What happens when multiple coder servers declare different time zones?

Reply:
I meant the site admin defines it before they can turn on the feature